### DISCUSSION TITLE
Problems encountered while using custom rules

### DISCUSSION THREAD

**Discussion by 6aink** [AdguardTeam/AdGuardHome#5892] [Q&A] [answered]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

x86

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

v0.108.0-b.36

#### What did you do?
I use the following rules in the custom blocking rules

||*^$dnstype=HTTPS

I found that its interception code is NODOMAIN.
Can I make its intercepts return NOERROR
I have tried all the settings, but I can't solve this error. Returning NODOMAIN will cause the iPhone to heat up abnormally.

**Comment by ameshkov** [comment, marked as answer]
Sure, you should use `$dnsrewrite` for that: https://adguard-dns.io/kb/general/dns-filtering-syntax/#dnsrewrite

Something like that:
`||*^$dnstype=HTTPS,dnsrewrite=NOERROR;;`

**Reply by 6aink** [reply]
Thank you very much for helping me solve this problem. I have also tried to use this method. I am missing one behind;. So it has been unable to work, thank you very much!
